### PR TITLE
Peddy

### PR DESCRIPTION
**Pull request by sima-r**
## PR checklist
This PR adds peddy module and peddy_check subworkflow into the pipeline. 
To Dos (maybe in another PR): add more functions in the subworkflow inorder to parse the output files and make reports or emails in case of abnormalities ? 
- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR> -stub`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test2samples,docker --outdir <OUTDIR> -stub`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by sima-r** [comment]
For some reason peddy_check subworkflow does not get initiated when running the pipeline. 😞

File: conf/modules.config [hunk 1]
```diff
@@ -1092,3 +1092,17 @@ process {
...
     }
 }
+
+//
+// PEDDY CHECK
+//
+
+process {
+    withName: '.*PEDDY_CHECK:PEDDY' {
+        ext.args = ''
```
Comment (jemten) [inline conf/modules.config:1102]: Peddy requires the argument `--sites hg38` when running grch38. We should be able to make the switch automatically by using the same logic as for vep
https://github.com/nf-core/raredisease/blob/079c91dacf0781e741cbd56085e3def17cd79e49/conf/modules.config#L715

File: modules/nf-core/peddy/main.nf [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    tuple val(meta), path("*.peddy.ped"), emit: ped
+    tuple val(meta), path("*.png")      , optional: true, emit: png
+    path "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
```
Comment (jemten) [inline modules/nf-core/peddy/main.nf:26]: `prefix` doesn't seem to be used in the module. 
I would add the line `--prefix $prefix` to the peddy command.

**Review by jemten — COMMENTED** [review]
Some comments :)

**Comment by Gwennid** [comment]
We created a new branch "peddy_backup" because there was an issue with that branch.

**Pull request closed without merging**